.travis.yml

```diff
@@ -4,6 +4,15 @@ rust:
 branches:
   only:
     - bench
+before_install:
+  - curl -L https://downloads.sourceforge.net/project/gnuplot/gnuplot/5.0.6/gnuplot-5.0.6.tar.gz > gnuplot-5.0.6.tar.gz
+  - tar xzf gnuplot-5.0.6.tar.gz
+  - cd gnuplot-5.0.6
+  - ./configure
+  - make
+  - sudo make install
```

Comment:
Maybe just install to some user location instead.

https://docs.travis-ci.com/user/migrating-from-legacy/